tests/toranj/start.sh

```diff
@@ -94,11 +94,11 @@ case $TORANJ_POSIX_APP_RCP_MODEL in
 esac
 
 if [ "$use_posix_app_with_rcp" = "no" ]; then
-    ./build.sh ${coverage_option} ncp || die
+    ./build.sh ${coverage_option} ncp --enable-address-sanitizer || die
```

Comment:
Should we only include `--enable-address-sanitizer` on Travis CI builds? Also, it looks like this needs to be plumbed through `tests/toranj/build.sh`.

@abtink thoughts?

Reply:
Fully agree. We should only enable it on travis builds. I use the `build.sh` script on my local machine to build a certain configuration quickly (don't want address-sanitizer enabled by default).

Also agree, it should be plumed through to `build.sh`.

Reply:
I'd leave address sanitizer always on in `tests/toranj/start.sh` since I think it's desired to have the test script checking whether there are memory hazards. In the `build.sh` I'll turn address sanitizer into an option disabled by default.